README.md

```diff
@@ -8,3 +8,6 @@ dependency management.
 ```sh
 pipenv sync # Installs all depedencies.
 ```
+
+## TODO's
+Change the way errors raised in process of running the script are handled. 
```

Comment:
We can make this a list by prepending each line/item with an asterisk. See https://guides.github.com/features/mastering-markdown/